chickadee/processes/wps_BCCAQ.py

```diff
@@ -87,30 +95,32 @@ def _handler(self, request, response):
         )
         climdown = get_package("ClimDown")
 
-        # Set parallelization
         log_handler(
             self,
             response,
-            "Setting parallelization",
+            "Setting R options",
             logger,
             log_level=loglevel,
-            process_step="parallelization",
+            process_step="set_R_options",
         )
-        doPar = get_package("doParallel")
-        doPar.registerDoParallel(cores=num_cores)
+        # Uses general_options_input
+        set_general_options(*args[6:12])
+        # Uses ca_options_input
+        set_ca_options(*args[12:19])
+        # Uses qdm_options_input
+        set_qdm_options(*args[19:])
```

Comment:
I agree that these are very handy. We should consider a better way to handle the args though. Passing them in this way can be difficult to read and follow. This is connected with my point above.

Reply:
@cairosanders I implemented `collect_args` to return `OrderedDict` with keys being identifiers of the inputs for versatility. Maybe you could write a `collect_args_wrapper` function using the keys?